snap/snapcraft.yaml

```diff
@@ -218,7 +218,7 @@ parts:
     after:
       - bess
     source: https://github.com/omec-project/upf.git
-    source-tag: v1.4.0
+    source-tag: v1.5.0
```

Comment:
Has anyone tested this with the rest of the core? 
Most changes between `1.4.0` and `1.5.0` are just bumping the dependencies, but I think it would be good to have it tested. 
We should probably also create similar PRs for the UPF rocks, shouldn't we?